.travis.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         documentation/bin/createDocu.sh vars documentation/docs-tmp/steps
         docker run --rm -it -v ${TRAVIS_BUILD_DIR}:/docs -w /docs/documentation squidfunk/mkdocs-material:3.0.4 build --clean --verbose --strict
     - name: Consumer Tests for s4sdk pipeline
+      if: repo = "SAP/jenkins-library" AND branch =~ /^master$|^integration\/.*$/
```

Comment:
Seems to work like it should. Additional ask: Can we also add another shorter prefix? I'd like something like `it/` as a prefix for "run integration tests on this". I think this makes sense because then we have a clear line between what `integration/*` means (integrating a branch from a fork) and `it/*` as PR on upstream.

Everyone fine with that?

@benhei @daniel-kurzynski @marcusholl

Reply:
>Additional ask: Can we also add another shorter prefix?

Technically adding another prefix is a no-brainer.
But it is difficult for me to understand the difference between `integration/*` and `it/*`. The semantics behind it is in both cases as far as I understand: "dear travis, please run the integration tests ..."

Reply:
Yes, that is right, but I have the slight fear that my `integration/feature-x` branch might be force pushed by accident one day by some one applying a PR change from a fork. Besides, I will have to type this over and over again, as it is my intention to always run IT in my branches, so the length of the prefix matters to me.

Reply:
Still not got the reason why we need both `integration/*` and `it/*`. I got the point with the short prefix. What about having only the short prefix?

> force push by accident.

Why is a force push less likely when we have `it` and `integration`. The human factor we have always ...

Reply:
My idea was to use the prefixes for those distinct purposes to minimize the human error chance. But I'm also fine with just `it`.

Reply:
shorted to `it`.

>those distinct purposes

I still not able to identify two distinct use cases For me there is only one: "Dear travis, would you please be so kind to run the integration tests." Basically I would like to understand this ...

Reply:
Purpose 1:

Integrate commits from a PR created from a fork. Workflow is that a core-team member (force) pushes the branch from the fork-remote to `integration/pr-xy`.

Purpose 2:

I as a core team member want to create a PR and wish for early feedback from the integration test, so I directly create a branch called `it/my-feature-x` and push it to the `SAP/jenkins-library` repo.

Does that convey my thoughts?